src-web/components/kappnav/common/NamespaceDropdown.js

```diff
@@ -18,17 +18,18 @@
  
 'use strict'
 
-import React from 'react'
-//import { connect } from 'react-redux'
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import lodash from 'lodash'
 import { DropdownV2 } from 'carbon-components-react'
-import msgs from '../../../../nls/kappnav.properties'
+import msgs from '../../../../nls/kappnav.properties';
+import { fetchLoadingSelectedNamespace } from '../../../reducers/BaseServiceReducer';
```

Comment:
We should be consistent on whether we end a line with `;` or not.  The future enablement of ESLint will flag the inconsistency.